deepchem/utils/batch_utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+Utility Functions for computing features on batch.
+"""
+import numpy as np
+
```

Comment:
Add a new section to docs/ for batch utils. Add this function and check the docs render correctly

Reply:
Done